src/dashboard/apps/consent/templates/consent/includes/_manage_consents.html

```diff
@@ -94,7 +94,15 @@
                   <td>
                     {% for station_name, ids in consent.stations_grouped.items %}
                       {% if forloop.counter0 > 0 %}<br />{% endif %}
-                      {{ station_name }} {% if ids %}({{ ids|join:", " }}){% endif %}
+                      {{ station_name }}
+                      {% if ids %}
+                        (
+                        {% for id in ids %}
+                          {{ id }}{% if not forloop.last %}, {% endif %}
+                          {% if forloop.counter|divisibleby:3 and not forloop.last %}<br>{% endif %}
```

Comment:
Is this still something to close simple tags?


```suggestion
                          {% if forloop.counter|divisibleby:3 and not forloop.last %}<br/>{% endif %}
```